### ISSUE TITLE
Replace IfBooleanTF (resp. If(No)ValueTF) by IfBooleanT (resp. If(No)ValueT) when possible?

### ISSUE THREAD

**Issue by dbitouze** [T-F-S/tcolorbox#114]
See:

https://github.com/T-F-S/tcolorbox/blob/78a915c1b1971a82cd3cce09f6008ccc065ab625/doc/latex/tcolorbox/tcolorbox.doc.xparse.tex#L166

https://github.com/T-F-S/tcolorbox/blob/78a915c1b1971a82cd3cce09f6008ccc065ab625/doc/latex/tcolorbox/tcolorbox.doc.xparse.tex#L274-L275

https://github.com/T-F-S/tcolorbox/blob/78a915c1b1971a82cd3cce09f6008ccc065ab625/doc/latex/tcolorbox/tcolorbox.doc.xparse.tex#L342

https://github.com/T-F-S/tcolorbox/blob/78a915c1b1971a82cd3cce09f6008ccc065ab625/doc/latex/tcolorbox/tcolorbox.doc.xparse.tex#L573

**Comment by T-F-S** [comment]
Yes, I should do that for didactical reasons.

Also, stupidly, I made some mistakes in implementing `IfBoolean(TF)`. The correct code is:

```latex
\tcbset{%
  IfBooleanTF/.code n args={3}{\IfBooleanTF{#1}{\pgfkeysalso{#2}}{\pgfkeysalso{#3}}},
  IfBooleanT/.code n args={2}{\IfBooleanT{#1}{\pgfkeysalso{#2}}},
  IfBooleanF/.code n args={2}{\IfBooleanF{#1}{\pgfkeysalso{#2}}},
}

```

**Comment by T-F-S** [comment]
Changed (and fixed) with version 4.40 (2020/09/25).
